Source/WebKit/UIProcess/API/glib/WebKitUserContentManager.cpp

```diff
@@ -224,12 +247,64 @@ void webkit_user_content_manager_remove_all_scripts(WebKitUserContentManager* ma
     manager->priv->userContentController->removeAllUserScripts();
 }
 
+/**
+ * webkit_user_script_message_get_value:
+ * @message: A #WebKitUserScriptMessage
+ *
+ * Return the message sent in by user.
+ *
+ * Returns: (transfer none): The #JSCValue sent by JavaScript code to the
+ * script message handler.
+ *
+ * Since: 2.38
+ */
+JSCValue* webkit_user_script_message_get_value(WebKitUserScriptMessage* userScriptMessage)
+{
+    if (!userScriptMessage->message)
+        return jsc_value_new_undefined(SharedJavaScriptContext::singleton().getOrCreateContext());
+
+    return userScriptMessage->message;
```

Comment:
This is a smart pointer, to return the raw `WebKitUserScriptMessage*` pointer we need to call `.get()` on it:

 ```suggestion
    return userScriptMessage->message.get();
```